evm/verify-contract.js

```diff
@@ -33,26 +33,22 @@ async function processCommand(config, chain, options) {
     printInfo('Contract address', options.address || chain.contracts[contractName]?.address);
 
     const contractConfig = chain.contracts[contractName];
+    const contractJson = getContractJSON(contractName);
+    let contract;
+    const contractFactory = await getContractFactoryFromArtifact(contractJson, wallet);
+    const excludedContracts = ['InterchainProposalSender', 'BurnableMintableCappedERC20', 'TokenManagerProxy', 'AxelarAmplifierGateway'];
```

Comment:
this causes duplication of case-specific handling and should be avoided (otherwise we need to think of updating 2 places)